src/main/resources/org/jenkinsci/plugins/docker/workflow/Docker.groovy

```diff
@@ -75,17 +75,15 @@ class Docker implements Serializable {
         new Image(this, id)
     }
 
-    String shell() {
-        node {
```

Comment:
This `node` call was also unnecessary—always triggered https://github.com/jenkinsci/docker-workflow-plugin/blob/b174d46226ef1095903f2e789355a3b216b46dda/src/main/resources/org/jenkinsci/plugins/docker/workflow/Docker.groovy#L64-L66